Strata/Languages/Boogie/Boogie.lean

```diff
@@ -34,9 +34,10 @@ namespace Boogie
    types.
 -/
 
-def typeCheck (options : Options) (program : Program) := do
+def typeCheck (options : Options) (program : Program) : Except Std.Format Program := do
   let T := { Lambda.TEnv.default with functions := Boogie.Factory, knownTypes := Boogie.KnownTypes }
-  let (program, _T) ← Program.typeCheck T program
+  let (program, T) ← Program.typeCheck T program
```

Comment:
Does this lead to a warning? I'd like to avoid increasing the number of warnings generated.